composer.json

```diff
@@ -17,11 +17,11 @@
     ],
 
     "require": {
-        "php": "^5.5.9|^7.0.8|^7.1.3|^7.2.5",
+        "php": "^5.5.9|^7.0.8|^7.1.3|^7.2.5|^8.0",
         "symfony/http-foundation": "~3.0|~4.0|~5.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.8 || ^5.0"
+        "phpunit/phpunit": "^4.8 || ^5.0 || ~9.0"
```

Comment:
Is this really working? This usually breaks all the tests.

Reply:
I mean.. let's try ? It's still a draft

Reply:
Sure thing.
I was simply reacting to your introduction stating there was no tests failure :)

Reply:
> Sure thing.
> I was simply reacting to your introduction stating there was no tests failure :)

I'm doing a lot of Github Action CI these days.. one more isn't a big deal